packages/mobile/src/components/inbox-unavailable-drawer/InboxUnavailableDrawer.tsx

```diff
@@ -105,129 +105,149 @@ const useStyles = makeStyles(({ spacing, typography, palette }) => ({
   }
 }))
 
-const mapActionToContent = (
+const useGetDrawerContent = (
```

Comment:
is this not just a component with props? 👀

Reply:
wow you're so right